ooni/ui/web/client/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  </head>
+  <body ng-app="app" ng-strict-di ng-cloak>
+
+    <app>
+      Loading...
```

Comment:
So, if I recall correctly, when one sees this page it means that the web-ui was not correctly installed... would it make sense, then, to state that here and add instructions how to get the ui?